test/powershell/Modules/Microsoft.PowerShell.Utility/Start-Sleep.Tests.ps1

```diff
@@ -17,6 +17,13 @@ Describe "Start-Sleep DRT Unit Tests" -Tags "CI" {
         $watch.Stop()
         $watch.ElapsedMilliseconds | Should BeGreaterThan 950
     }
+
+    It "Should work properly when sleeping with ms alias" {
+        $watch = [System.Diagnostics.Stopwatch]::StartNew()
+        Start-Sleep -ms 1000
+        $watch.Stop()
+        $watch.ElapsedMilliseconds | Should BeGreaterThan 950
```

Comment:
If we're worried about parameter's aliases, we could make a common test for all of them, just like we did for the cmdlets and their aliases as < cmdlet, parameter, alias, platform(?) >.